tools/modify_mesh_mask/README

```diff
@@ -0,0 +1,64 @@
+mesh_mask_modifier is a tool that modifies mesh_mask files. It reads a
+mesh_mask file and outputs a modified copy of the same file.
+
+Files involved
+--------------
+python/ctsm/mesh_mask_modifier.py
+python/ctsm/modify_mesh_mask.py
+tools/modify_mesh_mask/mesh_mask_modifier
+tools/modify_mesh_mask/modify_template.cfg
+
+Instructions
+------------
+To run on Cheyenne/Casper/Izumi
+1) (Un)load, execute, and activate the following:
+module unload python
+module load conda
+./manage_python_env
```

Comment:
@ekluzek another item to confirm when we go over this PR:
Should all the README documentation change to say ./py_env_create instead of ./manage_python_env?

Reply:
Yes.